javagen/src/main/java/com/azure/autorest/Javagen.java

```diff
@@ -97,9 +97,8 @@ protected NodeTuple representJavaBeanProperty(Object javaBean, Property property
 
             // Service client builder
             String builderPackage = ClientModelUtil.getServiceClientBuilderPackageName(client.getServiceClient());
-            String builderSuffix = ClientModelUtil.getBuilderSuffix();
-            javaPackage.addServiceClientBuilder(builderPackage,
-                client.getServiceClient().getInterfaceName() + builderSuffix, client.getServiceClient());
+            String builderClassName = ClientModelUtil.getBuilderClassName(client.getServiceClient().getInterfaceName());
+            javaPackage.addServiceClientBuilder(builderPackage, builderClassName, client.getServiceClient());
```

Comment:
What is the difference?

Reply:
CI forces builder name must end with "ClientBuilder". Old code appends "Builder". I add a check if it does not end with "Client", then append "ClientBuilder".

Reply:
This might cause breaking changes to existing packages, many of which the Builder lives in `implementation` package and hence not need to have the `Client`.

Would it possible to only apply this logic to `shouldGenerateSyncAsyncClients` or just `isLowLevelClient`?

Reply:
We can also specify `--title=PurviewScanningClient` to ignore name defined in swagger.

Reply:
Yeah, this might be easier.

Reply:
I'll revert this commit.

Reply:
Done